### DISCUSSION TITLE
Can't control HS fan and Chipset Fan

### DISCUSSION THREAD

**Discussion by PaCz1000** [Rem0o/FanControl.Releases#2354] [Q&A]
Hi guys,

I am not able to control the HS fan and Chipset fan on my Asus Crosshair Impact viii x570. I have installed the HWinfo plugin and I can view the RPM of these fans but I can't control them. I have also tried the Asus WMI plugin but it doesn't seem to work with my motherboard. Please can someone assist me?

**Comment by Rem0o** [comment]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/2812a581-4b89-4572-9d97-14e3978ee1e1)

Did you enable the Asus embedded EC source?

**Reply by PaCz1000** [reply]
Hi there, yes indeed, that was enabled from the get go.
